i18n.go

```diff
@@ -26,43 +24,50 @@ func newI18n(opts ...Option) {
 		ins.getLngHandler = defaultGetLngHandler
 	}
 
-	atI18n = ins
+	return ins
 }
 
 // Localize ...
 func Localize(opts ...Option) gin.HandlerFunc {
-	newI18n(opts...)
+	atI18n := newI18n(opts...)
 	return func(context *gin.Context) {
+		context.Set("i18n", atI18n)
```

Comment:
Because the length of the data in context is limited, in my opinion. We shouldn't do this.